cmd/minishift/cmd/addon/addon_install_test.go

```diff
@@ -58,31 +60,23 @@ func Test_install_with_enable_flag_works(t *testing.T) {
 	defer cli.TearDown(tmpMinishiftHomeDir, tee)
 
 	addOnManager := GetAddOnManager()
-	if len(addOnManager.List()) != 0 {
-		t.Fatal(fmt.Sprintf("There should be no add-ons installed. Got got : %v", addOnManager.List()))
-	}
+
+	assert.Equal(t, len(addOnManager.List()), 0)
```

Comment:
-1 assert.Len, we should use the correct assertions. There is more than just assert.Equals. This is in particular true for tests asserting errors (which we don't see here).

Reply:
Sure, will use assert.Len()